### PR TITLE
Excise openff dependency from openmm testing

### PR DESCRIPTION
**Pull request by orionarcher**
This PR removes the openff dependency from the openmm test suite. This conflict originally arose in PR #923.

### CODE REVIEW

**Comment by orionarcher** [comment]
@janosh would you mind taking a look?

File: src/atomate2/openmm/utils.py [hunk 1]
```diff
@@ -2,16 +2,28 @@
...
+from emmet.core.openmm import OpenMMInterchange
+from openmm import LangevinMiddleIntegrator, XmlSerializer
+from openmm.app import PDBFile
+
 if TYPE_CHECKING:
+    from contextlib import suppress
+
     from emmet.core.openmm import OpenMMTaskDocument
 
+    with suppress(ImportError):
```
Comment (janosh) [inline src/atomate2/openmm/utils.py:24]: `TYPE_CHECKING` is always `False` at run time so no need to `suppress` errors here. also, `suppress` is quite slow compared to `try/except` so would usually opt for that

Comment (orionarcher) [reply src/atomate2/openmm/utils.py:24]: I initially did try/except but it was caught by the linter. should I override then?

**Comment by orionarcher** [comment]
All fixed, thanks for review @janosh

File: src/atomate2/openmm/utils.py (hunk 1 above)
Comment (janosh) [reply src/atomate2/openmm/utils.py]: yes, we should ignore that `ruff` rule that recommends `contextlib.suppress`. bad rule imo

**Review by janosh — APPROVED** [review]
👍

**Review by janosh — APPROVED** [review]
looks great! 👍 thanks for the quick turnaround on https://github.com/materialsproject/atomate2/pull/923#issuecomment-2377150112 @orionarcher

**Pull request merged**